### ISSUE TITLE
Kobe plot colors for accessibility

### ISSUE THREAD

**Issue by MOshima-PIFSC** [nmfs-ost/satf#19]
The typical background colors for the quadrants of  a Kobe plot are red, yellow, green, and sometimes orange. These aren't great for accessibility and better colors/shades should be explored.  Colors I've used before are "palegreen", "khaki1", and "salmon" but there are probably better options for contrast.

**Comment by sbreitbart-NOAA** [comment]
I like [the Okabe-Ito color palette](https://clauswilke.com/dataviz/color-pitfalls.html#fig:palette-Okabe-Ito). What do you think about choosing four of these colors, like the first four? The example shows that they have good contrast someone with color-vision deficiency.
